NotificationsMessage.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime, timedelta
+import pytz, uuid
+
+class NotificationsPipelineHistory(object): # Pipeline History Object
+    def __init__(self, step, event):
+        self.step = step
+        self.event = event
+        self.timestamp = pytz.utc.localize(datetime.utcnow())
+    def __repr__(self):
+        return '<%s %s: %s>' %(self.step, self.event, self.timestamp)
+
+class NotificationsMessage(object):
+
+    def __init__ (self, name="edx.notifications.forums.post.created", fields=None, recipients=None):
```

Comment:
Made this change. The composer instead creates a NotificationsMessage object with this name now.